Gemfile

```diff
@@ -27,6 +35,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 #new gems added
 gem 'devise'
+gem 'pry-rails'
```

Comment:
@remyamuraleedharan i guess if you can add gem 'pry-rails',  to  :group => :development, which is not used in production.